CleverPush/Source/CleverPushInstance.m

```diff
@@ -1494,6 +1494,18 @@ - (void)handleNotificationOpened:(NSDictionary*)payload isActive:(BOOL)isActive
     if (!channelId) { // not init
         pendingOpenedResult = result;
     }
+
+    if (notification != nil && [notification objectForKey:@"url"] != nil && [[notification objectForKey:@"url"] length] != 0 && ![[notification objectForKey:@"url"] isKindOfClass:[NSNull class]]) {
+        NSURL *url = [NSURL URLWithString:[notification objectForKey:@"url"]];
+        if ([notification objectForKey:@"autoHandleDeepLink"] != nil && ![[notification objectForKey:@"autoHandleDeepLink"] isKindOfClass:[NSNull class]] && [[notification objectForKey:@"autoHandleDeepLink"] boolValue]) {
+            if ([[UIApplication sharedApplication] canOpenURL:url]) {
+                [[UIApplication sharedApplication] openURL:url options:@{} completionHandler:nil];
+            } else {
+                [CPUtils openSafari:url];
```

Comment:
I think this else condition is not required. If `canOpenURL` will return false, openSafari will also not work.